README.md

```diff
@@ -353,7 +377,8 @@ There are several options that are supported on the main app and subcommands. Th
 -   `.got_subcommand(App_or_name)`: Check to see if a subcommand was received on the command line.
 -   `.get_subcommands(filter)`: The list of subcommands given on the command line.
 -   `.get_parent()`: Get the parent App or nullptr if called on master App.
--   `.get_option(name)`: Get an option pointer by option name
+-   `.get_option(name)`: Get an option pointer by option name will throw if the specified option is not available,  nameless subcommands are also searched
+-   `.get_option_no_throw(name)`: Gen an option pointer by option name, this function will return a `nullptr` instead of throwing if the option is not available.
```

Comment:
```suggestion
-   `.get_option_no_throw(name)`: Get an option pointer by option name. This function will return a `nullptr` instead of throwing if the option is not available.
```